e2e/playwright/tests/meditor.spec.ts

```diff
@@ -0,0 +1,506 @@
+import { test, expect, type Page } from "@playwright/test";
+
+const clientUrl = "http://localhost:8085";
+
+const noLicenseSelector = async (page: Page) => {
+  await page.getByRole("tab", { name: "General Badge" }).click();
+  await expect(page.getByText("LicenseNo less than 1 items")).toBeVisible();
+};
+const noContributorSelector = async (page: Page) => {
+  await expect(
+    page.getByRole("tab", { name: "Dandiset contributors Badge" })
+  ).toBeVisible();
+};
+const invalidLicenseSelector = async (page: Page) => {
+  await page.getByRole("tab", { name: "General Badge" }).click();
+  await expect(
+    page.getByRole("button", { name: "License spdx:CC-BY-NC-" })
+  ).toBeVisible();
+};
+const titleTooLongSelector = async (page: Page) => {
+  await page.getByRole("tab", { name: "General Badge" }).click();
+  await expect(
+    page.getByText("Dandiset title150 characters maximum")
+  ).toBeVisible();
+};
```

Comment:
I think we should add some comments to describe what these selectors are actually doing (at the high level of "operations on the DANDI app").

I don't know if Playwright has general guidance for this sort of thing--it must be a common concern to mediate between the code that Playwright generates, and a general understanding of what actions triggered that generation.